### PR TITLE
Capacity count #177008898

### PR DESCRIPTION
**Pull request by bytheway875**
Screen Shot 2021-04-14 at 1 04 50 PM
Screen Shot 2021-04-14 at 1 07 09 PM
Screen Shot 2021-04-14 at 1 07 30 PM

### CODE REVIEW

**Review by embarnard — CHANGES_REQUESTED** [review]
Looks good! Only requested change is to add the Spanish translations

File: app/controllers/hub/organizations_controller.rb [hunk 1]
```diff
@@ -27,8 +27,9 @@ def show
 
     def index
       @coalitions = Coalition.accessible_by(current_ability).includes(:organizations)
-      @organizations = @vita_partners.organizations
-      @independent_organizations = @vita_partners.organizations.where(coalition: nil)
+      @vita_partners = @vita_partners.organizations.includes(:organization_capacity)
+      @organizations = @vita_partners
+      @independent_organizations = @vita_partners.organizations.includes(:organization_capacity).where(coalition: nil)
```
[the remark this answers is not in the document]
Comment (bytheway875) [reply app/controllers/hub/organizations_controller.rb:32]: yes! Good catch. I can't even remember what I was doing here, but It looks like there's an incomplete thought happening here.

File: app/views/hub/metrics/index.html.erb [hunk 2]
```diff
@@ -27,20 +27,23 @@
           <th scope="col" class="sticky index-table__header sortable" id="organization-name" width="40%" attr-direction="asc">
             Organization Name
           </th>
+          <th scope="col" class="index-table__header sortable" id="capacity-percentage">
+            Capacity %<sup>1</sup>
```
[the remark this answers is not in the document]
Comment (bytheway875) [reply app/views/hub/metrics/index.html.erb:31]: There's a little "legend" sticky at the bottom of the page to explain what is included in the calculation of each metric. It's a bit too long to comfortably fit in a tooltip.

File: config/locales/es.yml [hunk 3]
```diff
@@ -381,6 +381,10 @@ es:
         site_count:
           one: "%{count} sitio"
           other: "%{count} sitios"
+      form:
```
[the remark this answers is not in the document]
Comment (bytheway875) [reply config/locales/es.yml:384]: No! I need to use google translate. Good catch. Thanks!

File: app/javascript/lib/metrics_table_sort.js [hunk 4]
```diff
@@ -2,12 +2,17 @@ import { initSortableColumn } from './table_sort';
 
 // Sum site-level counts into org-level values for org row.
 function setupOrgLevelCounts() {
+    let totalCapacityCount = 0;
```
[the remark this answers is not in the document]
Comment (bytheway875) [reply app/javascript/lib/metrics_table_sort.js:5]: if you go to hub/metrics, there's a little hidden metrics dashboard.

File: app/controllers/hub/organizations_controller.rb [hunk 5]
```diff
@@ -27,8 +27,8 @@ def show
 
     def index
       @coalitions = Coalition.accessible_by(current_ability).includes(:organizations)
-      @organizations = @vita_partners.organizations
-      @independent_organizations = @vita_partners.organizations.where(coalition: nil)
+      @organizations = @vita_partners.organizations.includes(:organization_capacity)
```
Comment (embarnard) [inline app/controllers/hub/organizations_controller.rb:30]: nice looks good!

File: config/locales/es.yml [hunk 6]
```diff
@@ -381,6 +381,10 @@ es:
...
+        allows_greeters: Permite saludadores
```
Comment (embarnard) [inline config/locales/es.yml:385]: yay!

**Review by embarnard — APPROVED** [review]
Approved! Good work 😄

**Pull request merged**